gecko_client/pchain_types.go

```diff
@@ -1,14 +1,14 @@
 package gecko_client
 
 // ============= Blockchain ====================
-type BlockchainCreationInfo struct {
+type UnsignedTransaction struct {
```

Comment:
In my branch I've actually renamed this to `UnsignedTransactionInfo` (techincally this isn't an unsigned transaction - actually a wrapper object around it - and I'm not a big fan of redundancy in codepaths - e.g. `UnsignedTransaction.UnsignedTx` looks weird to me). Can we do the same here?

Reply:
done